auth/src/ios/auth_ios.mm

```diff
@@ -154,6 +157,24 @@ void UpdateCurrentUser(AuthData *auth_data) {
   SetUserImpl(auth_data, user);
 }
 
+void CheckEmulator(AuthData *auth_data) {
+  // Use emulator as long as this env variable is set, regardless its value.
+  if (std::getenv("USE_AUTH_EMULATOR") == nullptr) {
```

Comment:
Just making sure, iOS also uses the environment variable?

Reply:
Right we can set the environment variable through xcode running setup.